src/scripts/main.js

```diff
@@ -1,3 +1,34 @@
 'use strict';
 
-// write code here
+const list = document.querySelector('ul').children;
+
+function sortList(params) {
+  const sort = [...params].sort((itemA, itemB) => {
+    const numA = toNumber(itemA);
+    const numB = toNumber(itemB);
+
+    return numB - numA;
+  });
+
+  for (let i = 0; i < sort.length; i++) {
+    list[i].before(sort[i]);
+  }
+
+  return list;
+}
+
+function toNumber(number) {
+  return number.dataset.salary.slice(1).split(',').join('');
+}
+
+function getEmployees(params) {
```

Comment:
The parameter name 'params' is not descriptive enough. Consider renaming it to 'listItems' to better describe its purpose.